internal/persistence/postgres/tx.go

```diff
@@ -0,0 +1,42 @@
+package postgres
+
+import (
+	"context"
+
+	"github.com/jackc/pgx/v4"
+	"github.com/kong/koko/internal/persistence"
+)
+
+type postgresTx struct {
```

Comment:
Thanks for pulling that out like I did with MySQL, def helps keep the code more readable.